backend/src/main/java/com/bakdata/conquery/models/forms/util/DateRangeSubDivider.java

```diff
@@ -0,0 +1,13 @@
+package com.bakdata.conquery.models.forms.util;
+
+import com.bakdata.conquery.models.common.daterange.CDateRange;
+
+import java.util.List;
+import java.util.Locale;
+
+public interface DateRangeSubDivider {
+
+    List<CDateRange> subdivideRange(CDateRange range);
```

Comment:
CDateSet ?

Reply:
Die DateRanges werden letztendlich in DateRestricting Nodes der einzelnen SubQueries gefüttert. Das wäre hier nicht sinnvoll eine komplexe Stuktur noch herumzubauen, um sie letztendlich wieder auszupacken.